core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/provision/clusterproxy/ClusterProxyService.java

```diff
@@ -147,6 +156,16 @@ private String getAccountId(Stack stack) {
         return asList(gatewayTunnel, knoxTunnel);
     }
 
+    private List<CcmV2Config> ccmV2Configs(Stack stack) {
+        InstanceMetaData primaryGateway = stack.getPrimaryGatewayInstance();
+        return List.of(ServiceFamilies.GATEWAY.getDefaultPort()).stream()
```

Comment:
as this is a single item, the stream and list is an overkill here

Reply:
CcmV2 Api expects List of CCMV2 Config, I will remove stream though in a follow up PR.

Reply:
Removed stream.